src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstance.kt

```diff
@@ -21,19 +23,34 @@ class LocalInstance private constructor(aem: AemExtension) : AbstractInstance(ae
     override lateinit var password: String
 
     var debugPort: Int = 5005
+    var debugAddress: String = ""
+    private val debugAddressWithPort: String
+        get() = when (debugAddress) {
+            "*" -> "0.0.0.0:$debugPort"
+            "" -> "$debugPort"
+            else -> "$debugAddress:$debugPort"
+        }
 
     @get:JsonIgnore
     val jvmOptsDefaults: List<String>
         get() = mutableListOf<String>().apply {
-            if (debugPort > 0) {
-                add("-Xdebug")
-                add("-Xrunjdwp:transport=dt_socket,address=$debugPort,server=y,suspend=n")
+            if (debugPort in 1..65535) {
+                add(jvmDebugOpt)
             }
             if (password != Instance.PASSWORD_DEFAULT) {
                 add("-Dadmin.password=$password")
             }
         }
 
+    @get:JsonIgnore
+    private val jvmDebugOpt: String
+        get() = when {
+            SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_9) ->
+                "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=$debugAddressWithPort"
```

Comment:
$debugHost:$debugPort

Reply:
As you can see java guys named that whole pair as address. Not only IP

Reply:
address is short for Socket Address :)